public/app/features/explore/Logs/Logs.tsx

```diff
@@ -597,455 +635,347 @@ class UnthemedLogs extends PureComponent<Props, State> {
-                    deduplicatedRows={dedupedRows}
-                    dedupStrategy={dedupStrategy}
-                    onClickFilterLabel={onClickFilterLabel}
-                    onClickFilterOutLabel={onClickFilterOutLabel}
-                    showContextToggle={showContextToggle}
-                    getRowContextQuery={getRowContextQuery}
-                    showLabels={showLabels}
-                    showTime={showTime}
-                    enableLogDetails={true}
-                    forceEscape={forceEscape}
-                    wrapLogMessage={wrapLogMessage}
-                    prettifyLogMessage={prettifyLogMessage}
-                    timeZone={timeZone}
-                    getFieldLinks={getFieldLinks}
-                    logsSortOrder={logsSortOrder}
-                    displayedFields={displayedFields}
-                    onClickShowField={this.showField}
-                    onClickHideField={this.hideField}
-                    app={CoreApp.Explore}
-                    onLogRowHover={this.onLogRowHover}
-                    onOpenContext={this.onOpenContext}
-                    onPermalinkClick={this.onPermalinkClick}
-                    permalinkedRowId={this.props.panelState?.logs?.id}
-                    scrollIntoView={this.scrollIntoView}
-                    isFilterLabelActive={this.props.isFilterLabelActive}
-                    containerRendered={!!this.state.logsContainer}
-                    onClickFilterString={this.props.onClickFilterString}
-                    onClickFilterOutString={this.props.onClickFilterOutString}
-                    onPinLine={this.onPinToContentOutlineClick}
-                    pinLineButtonTooltipTitle={this.state.pinLineButtonTooltipTitle}
-                  />
-                </InfiniteScroll>
-              </div>
-            )}
-            {!loading && !hasData && !scanning && (
-              <div className={styles.logRows}>
-                <div className={styles.noData}>
-                  <Trans i18nKey="explore.logs.no-logs-found">No logs found.</Trans>
-                  <Button size="sm" variant="secondary" onClick={this.onClickScan}>
-                    <Trans i18nKey="explore.logs.scan-for-older-logs">Scan for older logs</Trans>
-                  </Button>
-                </div>
+                  getFieldLinks={getFieldLinks}
+                  logsSortOrder={logsSortOrder}
+                  displayedFields={displayedFields}
+                  onClickShowField={showField}
+                  onClickHideField={hideField}
+                  app={CoreApp.Explore}
+                  onLogRowHover={onLogRowHover}
+                  onOpenContext={onOpenContext}
+                  onPermalinkClick={onPermalinkClick}
+                  permalinkedRowId={panelState?.logs?.id}
+                  scrollIntoView={scrollIntoView}
+                  isFilterLabelActive={props.isFilterLabelActive}
+                  containerRendered={!!logsContainerRef}
```

Comment:
This may be also worth revising. I think it will always be truthy (the ref is always created, only .current changes). But with latest changes to setting scrollIntoView only once the ref is received it may not be needed at all.